deebot_client/commands/json/auto_empty.py

```diff
@@ -23,11 +23,16 @@ class SetAutoEmpty(ExecuteCommand):
     NAME = "setAutoEmpty"
 
     def __init__(
-        self, enable: bool | None = None, frequency: Frequency | str | None = None
+        self, enable: bool | str | None = None, frequency: Frequency | str | None = None
```

Comment:
When is `enable` a `str`?

Reply:
Please pass them in as frequency param and not the enable one also we should not override the enable param if set. Only set it when it is None